magento/magento2
# Fix php error in profiler when using PHP 8.1

### Description (*)
This PR fixes possible deprecation error:

```
Deprecated: strpos(): Passing null to parameter 1 ($haystack) of type string is deprecated 
in Magento/Framework/Profiler/Driver/Standard/Stat.php:213
```

### Fixed Issues
1. Fixes magento/magento2#35307

### Manual testing scenarios (*)
1. Appy the [first commit from this PR](https://github.com/magento/magento2/pull/35321/commits/42d88187c54c4ae80468c34f1319de075a48ec75) and run unit test: `./vendor/bin/phpunit -c dev/tests/unit/phpunit.xml.dist lib/internal/Magento/Framework/Profiler/Test/Unit/Driver/Standard/StatTest.php`
2. You'll see an error during the test
3. Apply all other commits, and run the test again.

### Contribution checklist (*)
